src/lib/polkadotProvider/query/orders.js

```diff
@@ -2,6 +2,9 @@ import { queryServicesById } from './services'
 import { queryLabsById } from './labs'
 import { queryDnaSamples } from './geneticTesting'
 import { ethAddressByAccountId } from './userProfile'
+import localStorage from '@/lib/local-storage'
+import axios from 'axios'
+const baseUrl = "https://backend-dev.debio.network"
```

Comment:
maybe you missed this mas, please use env

Reply:
nice catch, thanks @ciniiia 👍🏼